ref_ts/views.py

```diff
@@ -221,7 +221,7 @@ def download_refts_resource_files(request, shortkey, *args, **kwargs):
             return response
 
         path = "bags/" + str(shortkey) + ".zip"
-        response_irods = download_bag_from_irods(request, path)
+        response_irods = download_bag_from_irods(request, path, use_async=False)
```

Comment:
@pkdash @zhiyuli Since this function ```download_refts_resource_files()``` in ```ref_ts\views.py``` has the ```@api_view(['GET'])``` decorator, it is called both from ref_ts resource landing page and from REST API, so we are running to the same dilemma as we did for generic download() function in django_irods, and the solution we end up with is to have a separate rest_download() view function just for REST_API url mapping. Either ref_ts has to do something similar to add a new view function such as rest_download_refts_resource_files() to pass in ```rest_call``` parameter a ```True``` value, and modify rest_call url mapping accordingly, or we just leave it as is, since ref_ts always does sync download, the only difference between rest_call and resource landing page call is very minor, a matter of whether to return styled content or not. I'd suggest to get this PR merged first to fix the regression issue caused by async bag download work first, then @zhiyuli can take on adding another ```rest_download_refts_resource_files()``` view function as needed later in ref_ts/views.py. Let me know what you think regarding merging this regression error fix first with existing issues to further worked on by @zhiyuli if needed.